configs/olcf/job_script.olcf.bash

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# comment out if using debug queue
+#PBS -q batch
+# comment in to get the debug queue (only available on Titan)
+##PBS -q debug
+# change number of nodes to change the number of parallel tasks
+# (anything between 1 and the total number of tasks to run)
+#PBS -l nodes=10
+#PBS -l walltime=1:00:00
+#PBS -A cli115
```

Comment:
it is also the account/project space we use for the conda environment. I think all ACME should have access to it.